res/controllers/Traktor-Kontrol-S4-MK2-hid-scripts.js

```diff
@@ -1138,6 +1175,36 @@ TraktorS4MK2.callbackLoopMove = function(field) {
   }
 }
 
+TraktorS4MK2.sendLoopSizeMessage = function(group, beats) {
+  var LoopSizeIndicators = {
+      "deck2" : [0x2B, 0x3A],
+      "deck1" : [0x1B, 0x2A],
+  }
+  start_signal = LoopSizeIndicators[group]
+
+  var sizesArray = {
+    64: [1,2,4,5,6,7,9,10,11,13],
+    32: [1,2,3,4,7,9,11,12,14,15],
+    16: [2,3,9,10,12,13,14,15],
+    8:  [9,10,11,12,13,14,15],
+    4:  [9,10,11,13],
+    2:  [9,11,12,14,15],
+  }
+  var packets = Object();
```

Comment:
I think it would be safer to declare this object inside the loop so there's no risk of stale data getting reused

Reply:
IMO using the `Object()` constructor is odd. I'd prefer an empty object literal `{}` instead. But this is a really minor point. If you disagree you can leave it as is.